lib/arux_app/api/account.rb

```diff
@@ -207,7 +155,7 @@ def delete_relationship
       protected
 
       def api_route
-        "#{self.class.server_uri}/api/v#{api_version}"
+        "#{self.class.api_uri}/api/v#{api_version}"
```

Comment:
I wonder if we should do instance methods that handle `self.class` since we're stepping through it in this way quite frequently

Reply:
Thats a good call. 

Do you think every module should have its uri methods setup like this (for consistency), even if the module is not currently using an instance method?

```
      def self.public_uri
        AruxApp::API.uri(subdomain: "cart")
      end

      def public_uri
        self.class.public_uri
      end
      
      def self.api_uri
        AruxApp::API.uri(subdomain: "cart.api")
      end

      def api_uri
        self.class.api_uri
      end
```

Reply:
I'm good either way, I think

Typically would avoid putting unused methods in just in case, but in a gem it probably makes a little more sense - whatever you pick would work for me